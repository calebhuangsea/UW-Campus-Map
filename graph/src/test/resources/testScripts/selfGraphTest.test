# Test add edge for one node
# Tests that a graph can be created
# a node can be added to the graph
# an edge can be added to 1 node

# Create a graph
CreateGraph graph1

# Add one node
AddNode graph1 n1

# Add an edge for one edge
AddEdge graph1 n1 n1 e1

# Print the node in the graph and the outgoing edges from n1
ListNodes graph1
ListChildren graph1 n1