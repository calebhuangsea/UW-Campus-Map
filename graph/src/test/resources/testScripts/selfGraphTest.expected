# Test add edge for one node
# Tests that a graph can be created
# a node can be added to the graph
# an edge can be added to 1 node

# Create a graph
created graph graph1

# Add one node
added node n1 to graph1

# Add an edge for one edge
added edge e1 from n1 to n1 in graph1

# Print the node in the graph and the outgoing edges from n1
graph1 contains: n1
the children of n1 in graph1 are: n1(e1)
